panel/widgets/input.py

```diff
@@ -327,17 +327,9 @@ def __init__(self, **params):
         super().__init__(**params)
         self._update_value_bounds()
 
-    @staticmethod
-    def _date_to_datetime(x):
-        if isinstance(x, date):
-            return datetime(x.year, x.month, x.day)
-
     @param.depends('start', 'end', watch=True)
     def _update_value_bounds(self):
-        self.param.value.bounds = (
-            self._date_to_datetime(self.start),
-            self._date_to_datetime(self.end),
-        )
+        self.param.value.bounds = (self.start, self.end)
```

Comment:
I don't follow why this is no longer needed. Can you explain?

Reply:
`_date_to_datetime` removes the time in a datetime object, e.g., this `isinstance(datetime.datetime.today(), datetime.date)` will return `True`.